lib/Parser.js

```diff
@@ -0,0 +1,60 @@
+class Parser {
+    constructor(args) {
+        this.parse(args);
```

Comment:
confused on the usage for this class + method. I didn't really check what it was doing/where it was called.
It's not usual to just call the parse method directly in the constructor. For this specific case, i'd suggest making parse method static maybe?
My main question here would be: how do you return the result? you can also just create your instance and call Parser.parse()